tests/services/test_files_service.py

```diff
@@ -204,14 +204,28 @@ def test_get_last_output_files_for_entity(self):
         last_output_files = files_service.get_last_output_files_for_entity(
             self.asset.id
         )
-        self.assertEqual(
-            last_output_files[str(geometry.id)][geometry_file.name]["revision"],
-            5
-        )
-        self.assertEqual(
-            last_output_files[str(cache.id)][cache_file.name]["revision"],
-            3
-        )
+
+        # test last geometry file revision
+        last_file = [
+            f
+            for f in last_output_files
+            if (
+                f['output_type_id'] == str(geometry.id)
+                and f['name'] == geometry_file.name
+            )
+        ][0]
```

Comment:
Can you find a more straightforward test? It's a good practice to put at least logic as possible in tests (otherwise you must test the test too).

Reply:
something like this ?
```python
        # test last geometry file revision
        last_output_files = files_service.get_last_output_files_for_entity(
            self.asset.id,
            output_type_id=str(geometry.id),
            name=geometry_file.name
        )
        self.assertEqual(len(last_output_files), 0)
        self.assertEqual(last_output_files[0]['revision'], 5)
```

Reply:
Yes!